src/api/app/views/webui/feeds/notifications.rss.builder

```diff
@@ -9,14 +9,24 @@ xml.rss version: '2.0' do
 
     @notifications.each do |notification|
       xml.item do
-        xml.title notification.event.subject
-        # We render the event_mailer view to 'description'
-        xml.description render(
-          template: "event_mailer/#{notification.event.template_name}",
-          layout: false,
-          formats: :text,
-          locals: { event: notification.event.expanded_payload }
-        )
+        if notification.notifiable
+          xml.title notification.title
+          xml.description render(
+            template: "notifications/#{notification.template_name}",
+            layout: false,
+            formats: :text,
+            locals: { notification: notification }
+          )
+        else
+          # TODO: This 'else' clause will be removed after all records from notifications are migrated
```

Comment:
Please mention `responsive_ux` somehow so we don't forget this.

Reply:
Done.